templates/cdk-service-kicker

```diff
@@ -4,11 +4,15 @@ set -eu
 # This service runs on boot to work around issues relating to LXD and snapd.
 
 # Workaround for https://github.com/conjure-up/conjure-up/issues/1448
-(set +e
-  echo "applying snapd apparmor profiles"
-  apparmor_parser /var/lib/snapd/apparmor/profiles/*
-  echo "apparmor_parser: exit status $?"
-)
+if [ -f '/proc/1/environ' ] && grep '^container=lxc' /proc/1/environ; then
```

Comment:
Super small nit pick, but there's some gobbledeegook output with grepping the (aparently) binary environ file:
```
root@juju-client-box-1:~# if [ -f '/proc/1/environ' ] && grep '^container=lxc' /proc/1/environ; then echo CONTAINERED; fi
Binary file /proc/1/environ matches
CONTAINERED
```

If you feel like it, get rid of the extra output with a `-q` to grep:
```
root@juju-client-box-1:~# if [ -f '/proc/1/environ' ] && grep -q '^container=lxc' /proc/1/environ; then echo CONTAINERED; fi
CONTAINERED
```

Reply:
I like the gobbledeegook. It gives it that special half-assed flavor that goes well with a service like cdk-service-kicker.

I'll add the `-q`, thanks :+1: